### PR TITLE
Add safecility to vendors and lighting controller product

### PR DESCRIPTION
**Pull request by mcallistertad**

### CODE REVIEW

File: vendor/safecility/emergency-light-controller.yaml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+# Valid values are: bluetooth, nfc, wifi, serial (when the user has a serial interface to set the keys)
+# and firmware (when the user should change the firmware to set the keys).
+keyProgramming:
+  - bluetooth
+
+# Key security (optional)
+# Valid values are: none, read protected and secure element.
+keySecurity: secure element
+
+# Product and data sheet URLs (optional)
```
Comment (Jaime-Trinidad) [inline vendor/safecility/emergency-light-controller.yaml:75]: hi @mcallistertad do you have your devices URLs? if not please remove these fields here, you left the example information. Thanks

File: vendor/safecility/emergency-light-controller-codec.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Uplink decoder decodes binary data uplink into a JSON object (optional)
+# For documentation on writing encoders and decoders, see: https://thethingsstack.io/integrations/payload-formatters/javascript/
+uplinkDecoder:
+  fileName: emergency-light-controller.js
```
Comment (Jaime-Trinidad) [inline vendor/safecility/emergency-light-controller-codec.yaml:4]: Hi @mcallistertad can you provide and example?

**Comment by Jaime-Trinidad** [comment]
Hi @mcallistertad thanks for your contribution please sign the cla

**Comment by johanstokking** [comment]
@mcallistertad This also needs a rebase to resolve conflicts

**Comment by Jaime-Trinidad** [comment]
Hi @mcallistertad any updates?

**Comment by Jaime-Trinidad** [comment]
Hello @mcallistertad we don't received any updates since may, can you sign CLA and make the updates. Thank you.

**Pull request closed without merging**